### ISSUE TITLE
Question for the feedback module on the documentation site

### ISSUE THREAD

**Issue by BasHamer** [Azure/azure-cli#13524]
**Is your feature request related to a problem? Please describe.**
I'd like to incorporate a similar module like your feedback module at the bottom of https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest for a documentation site. I was hoping to find the code for that. 

**Describe the solution you'd like**
A sample HTML page with the javascript to make the feedback section work so I can point it to another repo to reuse it.

**Describe alternatives you've considered**
View source, steal the HTML, didn't find the javascript so it is probably minified. Searched this repo for the feedback keyword and read some python code that looked unrelated.

**Additional context**
I really like the setup, hence why I'd like to use or copy it.

**Comment by yonzhan** [comment]
@fengzhou-msft please take a look

**Comment by BasHamer** [comment]
as an FYI we build it as a plugin. well, some of it. unfortunately, I'm not sure how we can mimic the data creation at this point.

the project is here http://feedback-on-github.possumlabs.com/ or https://github.com/Possum-Labs/Feedback-On-Github

**Comment by fengzhou-msft** [comment]
@jsntcy any idea how to add a feedback module like the doc site did?

**Comment by jsntcy** [comment]
@BasHamer, below is the code for "feedback" section in docs site
`<section class="feedback-section is-relative" data-bi-name="feedback-section">

    <h2 id="feedback" class="is-size-2 has-margin-top-large">{%- loc feedback -%}</h2>

    <div class="alert choose-feedback-type has-margin-top-small has-margin-bottom-none">
        <p aria-hidden="true" id="send-feedback-about" class="has-margin-top-none has-margin-bottom-none">{%- loc submitAndViewFeedbackFor -%}</p>

        <div class="choose-feedback-buttons has-margin-top-large has-margin-bottom-none">
            {%- if page._op_feedback_product_url != "" -%}
                <a class="button feedback-type-product has-margin-bottom-small" aria-label="{%- loc sendFeedbackAboutThisProduct -%}" href="{{page._op_feedback_product_url}}" data-bi-name="product-feedback">
                    <span>{%- loc thisProduct %}</span>
                    <span aria-hidden="true" class="icon docon docon-navigate-external is-size-2 has-margin-left-none"></span>
                </a>
            {%- endif -%}

			<a class="button feedback-type-product has-margin-bottom-small github-link" aria-label="{%- loc sendFeedbackAboutThisPage -%}" data-bi-name="create-issue-on-github">
				<span aria-hidden="true" class="docon docon-brand-github has-padding-right-extra-small"></span>
				<span>{%- loc thisPage %}</span>
			</a>
        </div>
    </div>

    <div class="action-container is-flex has-flex-justify-content-end has-margin-top-small has-margin-bottom-small">
        <a class="view-on-github" data-bi-name="view-on-github" href="https://github.com/{{page._op_feedback_github_repo}}/issues">
            <span aria-hidden="true" class="docon docon-brand-github"></span>
            <span>{% loc viewAllPageFeedback %}</span>
            <span aria-hidden="true" class="icon docon docon-navigate-external is-size-3"></span>
        </a>
    </div>
</section>
`

**Comment by BasHamer** [comment]
I'd like to include it on any documentation site. I like it and wanted to see if I could add it to the documentation that we were building for a package (http://www.aprentidigital.com/DSLDocumentation/).  I've found answers to questions in the azure documentation, and it is more inviting to leave feedback, and inherently reduces duplicate issues.

It looks like it is custom code and not a component, I worried about that with the GitHub authentication aspect.
